SocialCareCaseViewerApi.Tests/V1/Gateways/Database/CreatePersonalRelationshipTests.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Linq;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SocialCareCaseViewerApi.V1.Helpers;
+using SocialCareCaseViewerApi.V1.Gateways;
+using SocialCareCaseViewerApi.Tests.V1.Helpers;
+using Microsoft.EntityFrameworkCore;
+
+namespace SocialCareCaseViewerApi.Tests.V1.Gateways.Database
+{
+    [TestFixture]
+    public class CreatePersonalRelationshipTests : DatabaseTests
+    {
+        private DatabaseGateway _databaseGateway;
+        private Mock<IProcessDataGateway> _mockProcessDataGateway = new Mock<IProcessDataGateway>();
+        private Mock<ISystemTime> _mockSystemTime = new Mock<ISystemTime>();
+
+        [SetUp]
+        public void Setup()
+        {
+            _databaseGateway = new DatabaseGateway(DatabaseContext, _mockProcessDataGateway.Object, _mockSystemTime.Object);
+            DatabaseContext.ChangeTracker.QueryTrackingBehavior = QueryTrackingBehavior.NoTracking;
+        }
+
+        [Test]
+        public void CreatesAPersonalRelationship()
+        {
+            var (person, otherPerson) = PersonalRelationshipsHelper.SavePersonAndOtherPersonToDatabase(DatabaseContext);
+            var type = DatabaseContext.PersonalRelationshipTypes.FirstOrDefault(prt => prt.Description == "parent");
+            var request = PersonalRelationshipsHelper.CreatePersonalRelationshipRequest(
+                person.Id, otherPerson.Id, type.Id, type.Description
+            );
+
+            _databaseGateway.CreatePersonalRelationship(request);
+
+            var personalRelationship = DatabaseContext.PersonalRelationships.FirstOrDefault();
+            personalRelationship.PersonId.Should().Be(request.PersonId);
```

Comment:
for checks like this we might want to do

`personalRelationship?.PersonId.Should().Be(request.PersonId);`

This way if personalRelationships is null instead of `NullReferenceException` being thrown we will have the test fail because `null` is not equal to `request.PersonId`.

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and-

Reply:
Ooooooooo, I really like this! Thanks for the suggestion! 💯